src/main/groovy/com/todo/service/impl/TaskServiceImpl.groovy

```diff
@@ -0,0 +1,38 @@
+package com.todo.service.impl
+
+import com.google.inject.Inject
+import com.todo.dao.TaskDaoI
+import com.todo.db.sql.tables.pojos.Task
+import com.todo.service.TaskServicesI
+
+import java.sql.Connection
+
+class TaskServiceImpl implements TaskServicesI {
+
+    private final TaskDaoI TaskService
```

Comment:
Variable namings in Java/Groovy are camel case, hence it should be something like taskService